frontend/app/login/page.jsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+import { GoogleAuthProvider, signInWithRedirect } from "firebase/auth";
+import Cookies from "js-cookie";
+import { auth } from "../../firebase/config";
+import loginStyles from "../../styles/Login.module.css";
+
+const Login = () => {
+    const provider = new GoogleAuthProvider();
+
+    const signIn = async () => {
+        try {
+            Cookies.set('isLoggedIn', 'true');
+            await signInWithRedirect(auth, provider);
+        } catch(err) {
+            console.log(err);
+        }
+    }
+    
+    return (
+        <div className={loginStyles.parent}>
```

Comment:
I noticed elsewhere you are using `container` and here you are using `parent`. It's ideal to pick one convention and stick to it.